.gitignore

```diff
@@ -5,3 +5,10 @@ npm-debug.log
 package-lock.json
 .envrc
 *.code-workspace
+lerna-debug.log
+packages/jwt-verifier/okta-oidc-tck-*
```

Comment:
Instead of placing these at the global `.gitignore`, they can be placed inside of their respective subdirectories.

Reply:
This can be made simpler by just doing this:

```
# Ignore TCK-related files in all folders
okta-oidc-tck*
target
```